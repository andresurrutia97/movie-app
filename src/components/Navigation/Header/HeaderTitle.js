import React from 'react';
import PropTypes from 'prop-types';

const HeaderTitle = ({ title }) => {
  return (
    <span className="font-semibold flex justify-center custom-blue-text text-2xl">
      {title}
    </span>
  );
};

HeaderTitle.propTypes = {
  title: PropTypes.string.isRequired,
};

export default HeaderTitle;
