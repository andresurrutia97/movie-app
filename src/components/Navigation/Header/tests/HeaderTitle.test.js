import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import HeaderTitle from '../HeaderTitle';

describe('<HeaderTitle />', () => {
  const component = <HeaderTitle title="test" />;

  it('should render correctly', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should have title prop equal to "test"', () => {
    const wrapper = shallow(component);
    expect(wrapper.props().children).toEqual('test');
  });
});
