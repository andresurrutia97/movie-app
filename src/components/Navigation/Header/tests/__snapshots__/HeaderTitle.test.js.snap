// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderTitle /> should render correctly 1`] = `
<span
  className="font-semibold flex justify-center custom-blue-text text-2xl"
>
  test
</span>
`;
